radio/util/tts_ru.py

```diff
@@ -0,0 +1,163 @@
+                            (u"джи", "g2"),(u"джи", "g5"),
+                            (u"градусов", "degree0"), (u"градус", "degree1"),
+                            (u"градуса", "degree2"), (u"градусов", "degree5"),
+                            (u"радиан", "rad0"), (u"радиан", "rad1"),
+                            (u"радиана", "rad2"), (u"радиан", "rad5"),
+                            (u"миллилитров", "ml0"), (u"миллилитр", "ml1"),
+                            (u"миллилитра", "ml2"), (u"миллилитров", "ml5"),
+                            (u"унций", "founce0"), (u"унция", "founce1"),
+                            (u"унции", "founce2"), (u"унций", "founce5"),
+                            (u"часов", "hour0"), (u"час", "hour1"),
+                            (u"часа", "hour2"), (u"часов", "hour5"),
+                            (u"минут", "minute0"), (u"минута", "minute1"),
+                            (u"минуты", "minute2"), (u"минут", "minute5"),
+                            (u"секунд", "second0"), (u"секунда", "second1"),
+                            (u"секунды", "second2"), (u"секунд", "second5"),
+                            ]):
+    systemSounds.append((s, filename(f, PROMPT_SYSTEM_BASE + 115 + i)))
+
+for s, f, a in [(u"Триммер по центру", "midtrim", 495),
+                (u"Триммер максимум", "maxtrim", NO_ALTERNATE),
+                (u"Триммер минимум", "mintrim", NO_ALTERNATE),
+                (u"Таймер 1. Время вышло", "timovr1", NO_ALTERNATE),
+                (u"Таймер 2. Время вышло", "timovr2", NO_ALTERNATE),
+                (u"Таймер 3. Время вышло", "timovr3", NO_ALTERNATE),
+                (u"Аккумулятор садится!", "lowbatt", 485),
+                (u"Внимание, пульт не выключен!", "inactiv", 486),
+                (u"Проверь газ!", "thralert", 481),
+                (u"Проверь, тумблеры!", "swalert", 482),
+                (u"Ошибка памяти", "eebad", 483),
+                (u"Опен тэ, икс приветствует вас!", "hello", 480),
+                (u"Слабый сигнал!", "rssi_org", NO_ALTERNATE),
+                (u"Очень слабый сигнал!", "rssi_red", NO_ALTERNATE),
+                (u"Проблема с антэнной!", "swr_red", NO_ALTERNATE),
+                (u"Телеметрия потеряна", "telemko", NO_ALTERNATE),
+                (u"Телеметрия, восстановлена", "telemok", NO_ALTERNATE),
+                (u"Тренерский сигнал потерян", "trainko", NO_ALTERNATE),
+                (u"Тренерский сигнал восстановлен", "trainok", NO_ALTERNATE),
+                (u"Сенсор отключен", "sensorko", NO_ALTERNATE),
+                (u"Серва перегружена", "servoko", NO_ALTERNATE),
+                (u"Перегрузка", "rxko", NO_ALTERNATE),
+                (u"Приемник всё ещё подключен!", "modelpwr", NO_ALTERNATE),
+                ]:
+    systemSounds.append((s, filename(f, a)))
+
+for i, (s, f) in enumerate([
+                            (u"Аккумулятор садится!", "lowbat"),
+#                            (u"gear!, up!", "gearup"),
```

Comment:
Lazyness or a good reason not include the optional sounds? :)

Reply:
unfortunately, English texts sounds really weird, if I use Russian voice. So I decided to translate most important sounds, and later, after forum discussion, I'll update optional sounds.